pkg/component/component.go

```diff
@@ -833,6 +831,9 @@ func varsForPlatform(platform PlatformDetail) (*transpiler.Vars, error) {
 			"family":   platform.Family,
 			"major":    platform.Major,
 			"minor":    platform.Minor,
+			"install": map[string]interface{}{
+				"default": paths.ArePathsEqual(paths.Top(), paths.InstallPath(paths.DefaultBasePath)),
+			},
```

Comment:
I don't know if I like this nested inside of `runtime.*`. What about having install at the top. So it would be `install.default` versus `runtime.install.default`?

Reply:
I'm fine with changing the name.  I used `runtime.install.default` because that is what @cmacknz had in original issue (#2599), we should probably have him comment.

Personally, I have a slight preference for boolean variable names to sound more like questions, so something like `runtime.installed_in_default`, or `install.in_default`, but I'm fine with `install.default` too.

Reply:
Yeah I agree that `.default` is not the best name either. I would be good with `install.in_default`.

Reply:
No strong opinions on the name, just wanted to provide a starting point. `install.in_default` or `install.default` works.

Reply:
I changed name to `install._in_default`